roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   hostNetwork: true
 {% if kube_version | version_compare('v1.6', '>=')  %}
-  dnsPolicy: ClusterFirstWithHostNet
+  dnsPolicy: ClusterFirst
```

Comment:
For what it's worth, this is the same as 1.5 with "Default" policy. HostNet pods don't get cluster dns settings. They get the hosts's /etc/resolv.conf. Still approving, though :)